src/utils/googleCalendarUtil.js

```
@@ -61,20 +61,17 @@ class GoogleCalendarUtil {
         throw new Error('event type not supported');
       }
     }
-    await this.calendar.events.insert(
-      {
+    try {
+      await this.calendar.events.insert({
         auth: this.auth,
         calendarId: GOOGLE_CALENDAR_ID,
         resource: event,
-      },
-      function fn(err) {
-        if (err) {
-          // eslint-disable-next-line no-console
-          console.error('There was an error contacting the Calendar service: ' + err);
-          return;
-        }
-      },
-    );
+      });
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
+      throw new Error(error);
+    }
```

throws an error if the startDate comes after the endDate

---

google API will throw an error if the `startDate` is after the `endDate`. Previously, errors were silenced